src/index.html

```diff
@@ -3,12 +3,51 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <span class="logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </span>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones and phones" class="nav__link">смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a href="#laptops and computers"
+             class="nav__link"
+             data-qa="hover">
```

Comment:
```suggestion
            <a
              href="#laptops_and_computers"
              class="nav__link"
              data-qa="hover"
            >
```
Не используй пробелы в якорях
Форматируй теги с большим количеством атрибутов таким образом